proto/engine/v1/epbs.proto

```diff
@@ -0,0 +1,122 @@
+option php_namespace = "Ethereum\\Engine\\v1";
+
+
+message PayloadAttestationData {
+	bytes beacon_block_root = 1 [(ethereum.eth.ext.ssz_size) = "32"];
+	uint64 slot 		= 2 [(ethereum.eth.ext.cast_type) = "github.com/prysmaticlabs/prysm/v4/consensus-types/primitives.Slot"];
+	bool payload_present 	= 3;
+}
+
+message PayloadAttestation {
+  	bytes aggregation_bits          = 1 [(ethereum.eth.ext.ssz_size) = "ptc.size", (ethereum.eth.ext.cast_type) = "ptc.type"];
+	PayloadAttestationData data 	= 2;
+    	bytes signature 		= 3 [(ethereum.eth.ext.ssz_size) = "96"];
+}
+
+message PayloadAttestationMessage {
+	uint64 validator_index 		= 1 [(ethereum.eth.ext.cast_type) = "github.com/prysmaticlabs/prysm/v4/consensus-types/primitives.ValidatorIndex"];
+	PayloadAttestationData data 	= 2;
+    	bytes signature 		= 3 [(ethereum.eth.ext.ssz_size) = "96"];
+}
+
+message InclusionListSummaryEntry {
+	bytes address 	 = 1 [(ethereum.eth.ext.ssz_size) = "20"];
+	uint64 gas_limit = 2;
+}
+
+message InclusionListSummary {
+	uint64 proposer_index 		  	   = 1 [(ethereum.eth.ext.cast_type) = "github.com/prysmaticlabs/prysm/v4/consensus-types/primitives.ValidatorIndex"];
+	repeated InclusionListSummaryEntry entries = 2 [(ethereum.eth.ext.ssz_max) = "max_inclusion_list.size"];
+}
+
+message SignedInclusionListSummary {
+	InclusionListSummary message = 1;
+    	bytes signature 	     = 2 [(ethereum.eth.ext.ssz_size) = "96"];
+}
+
+message InclusionList {
+	SignedInclusionListSummary summary = 1;
+	uint64 slot 			   = 2 [(ethereum.eth.ext.cast_type) = "github.com/prysmaticlabs/prysm/v4/consensus-types/primitives.Slot"];
+	repeated bytes transactions 	   = 3 [(ethereum.eth.ext.ssz_size) = "?,?", (ethereum.eth.ext.ssz_max)  = "max_inclusion_list.size,1073741824"];
+}
+
+message ExecutionPayloadHeaderEPBS {
+	bytes parent_hash       		= 1 [(ethereum.eth.ext.ssz_size) = "32"];
+	bytes fee_recipient     		= 2 [(ethereum.eth.ext.ssz_size) = "20"];
+	bytes state_root        		= 3 [(ethereum.eth.ext.ssz_size) = "32"];
+	bytes receipts_root     		= 4 [(ethereum.eth.ext.ssz_size) = "32"];
+	bytes logs_bloom        		= 5 [(ethereum.eth.ext.ssz_size) = "logs_bloom.size"];
+	bytes prev_randao       		= 6 [(ethereum.eth.ext.ssz_size) = "32"];
+	uint64 block_number     		= 7;
+	uint64 gas_limit        		= 8;
+	uint64 gas_used       			= 9;
+	uint64 timestamp        		= 10;
+	bytes extra_data        		= 11 [(ethereum.eth.ext.ssz_max) = "extra_data.size"];
+	bytes base_fee_per_gas  		= 12 [(ethereum.eth.ext.ssz_size) = "32"];
+	bytes block_hash       			= 13 [(ethereum.eth.ext.ssz_size) = "32"];
+	bytes transactions_root 		= 14 [(ethereum.eth.ext.ssz_size) = "32"];
+	bytes withdrawals_root  		= 15 [(ethereum.eth.ext.ssz_size) = "32"];
+	uint64 blob_gas_used  			= 16;
+	uint64 excess_blob_gas  		= 17;
+	bytes inclusion_list_summary_root 	= 18 [(ethereum.eth.ext.ssz_size) = "32"];
+	bytes inclusion_list_exclusions_root 	= 19 [(ethereum.eth.ext.ssz_size) = "32"];
```

Comment:
This is primarily a spec detail, but using a root over a list of uint64 is silly. Is the concern exposing what the exclusion list where we shouldn't? Exclusion is mostly an implementation detail anyway, and we might change it later

Reply:
the root is only 32 bytes, the list can be larger